scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -51,98 +59,166 @@ include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+3]
+====
 
-include::modules/ztp-configuring-ipv6.adoc[leveloffset=+2]
+// End of 340 content
 
-include::modules/ztp-troubleshooting-the-managed-cluster.adoc[leveloffset=+2]
+include::modules/ztp-creating-the-policygentemplate-cr.adoc[leveloffset=+1]
 
+include::modules/ztp-policygentemplates-for-ran.adoc[leveloffset=+1]
 
-// RAN policies
+include::modules/ztp-generating-ran-policies.adoc[leveloffset=+1]
 
-include::modules/ztp-applying-the-ran-policies-for-monitoring-cluster-activity.adoc[leveloffset=+1]
+include::modules/ztp-creating-ztp-custom-resources-for-multiple-managed-clusters.adoc[leveloffset=+1]
 
-include::modules/ztp-applying-source-custom-resource-policies.adoc[leveloffset=+2]
+include::modules/ztp-installing-the-gitops-ztp-pipeline.adoc[leveloffset=+1]
 
-include::modules/ztp-the-policygentemplate.adoc[leveloffset=+2]
+include::modules/ztp-preparing-the-ztp-git-repository.adoc[leveloffset=+2]
 
-include::modules/ztp-things-to-consider-when-creating-custom-resource-policies.adoc[leveloffset=+2]
+include::modules/ztp-preparing-the-hub-cluster-for-ztp.adoc[leveloffset=+2]
+
+include::modules/ztp-deploying-a-site.adoc[leveloffset=+2]
+
+// 340 2/2
 
-include::modules/ztp-generating-ran-policies.adoc[leveloffset=+2]
+include::modules/ztp-deploying-additional-changes-to-clusters.adoc[leveloffset=+2]
 
+include::modules/ztp-using-pgt-to-update-source-crs.adoc[leveloffset=+2]
 
-// Cluster provisioning
+include::modules/ztp-adding-new-content-to-gitops-ztp.adoc[leveloffset=+2]
 
-include::modules/ztp-cluster-provisioning.adoc[leveloffset=+1]
+include::modules/ztp-talo-integration.adoc[leveloffset=+1]
 
-include::modules/ztp-machine-config-operator.adoc[leveloffset=+2]
+include::modules/ztp-customizing-the-install-extra-manifests.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-ptp-fast-events.adoc[leveloffset=+2]
 
-include::modules/ztp-performance-addon-operator.adoc[leveloffset=+2]
+include::modules/ztp-configuring-uefi-secure-boot.adoc[leveloffset=+2]
 
-include::modules/ztp-sriov-operator.adoc[leveloffset=+2]
+// End of 340
 
-include::modules/ztp-precision-time-protocol-operator.adoc[leveloffset=+2]
+// include::modules/ztp-creating-the-site-secrets.adoc[leveloffset=+2]
+// Remove this topic - keep the note and move to Step 3 in "Deploying a site"
+
+include::modules/ztp-monitoring-deployment-progress.adoc[leveloffset=+1]
+
+// Definition of done
+include::modules/ztp-definition-of-done-for-ztp-installations.adoc[leveloffset=+1]
+include::modules/ztp-creating-a-validator-inform-policy.adoc[leveloffset=+2]
+include::modules/ztp-querying-the-policy-compliance-status-for-each-cluster.adoc[leveloffset=+2]
+
+//Troubleshooting
+
+include::modules/ztp-troubleshooting-gitops-ztp.adoc[leveloffset=+1]
+
+include::modules/ztp-validating-the-generation-of-installation-crs.adoc[leveloffset=+2]
+
+include::modules/ztp-validating-the-generation-of-policy-crs.adoc[leveloffset=+2]
```

Comment:
This module is a duplicate of the next one. This one (ztp-validating-the-generation-of-policy-crs) is not accurate and should be removed.